.changeset/stupid-carpets-remain.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-catalog-backend-module-bitbucket-server': minor
+---
+
+Add new plugin catalog-backend-module-bitbucket-server which adds the `BitbucketServerEntityProvider`.
+
+The entity provider is meant as a replacement for the `BitbucketDiscoveryProcessor` to be used with Bitbucket Server (Bitbucket Cloud already has a replacement).
```

Comment:
As this will show up at the changelog, it would be great to add guidance for the migration from the processor to the new entity provider. A reference to the new documentation on top would be great, too.